README.md

```diff
@@ -193,6 +193,21 @@ mergeStream(project.sources(), project.dependencies())
   .pipe(gulp.dest('build/'));
 ```
 
+### Custom Elements ES5 Adapter
+
+If you are serving ES5 custom elements (either from source or by compilation), it is critical to include the [Custom Elements ES5 Adapter](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js). This adapter provides compatibility between custom elements defined as ES5-style classes and browsers with native implementations of the Custom Elements API, such as Chrome. See the [adapter documentation](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js) for details of why this is necessary.
+
+The `project.addCustomElementsEs5Adapter()` method will return a transform stream that identifies your entry point HTML file and injects a block of HTML that loads the adapter when it is required.
```

Comment:
Other comments:
- `entry point` -> `entrypoint`
- Does this stream now only change your entrypoint? Or does it still affect all files where import is found?